src/main/java/br/com/deveficiente/calendario/controller/validacao/TipoNotificacaoValido.java

```diff
@@ -0,0 +1,26 @@
+package br.com.deveficiente.calendario.controller.validacao;
+
+import br.com.deveficiente.calendario.controller.form.EventoForm;
+import br.com.deveficiente.calendario.controller.form.NotificacaoForm;
+import br.com.deveficiente.calendario.model.enums.TipoNotificacao;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+
+public class TipoNotificacaoValido implements Validator {
```

Comment:
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*

Se receber como enum lá, esse validador já perde a necessidade de existir :)